src/program.ts

```diff
@@ -28,7 +28,7 @@ export function run<Init, State, Action, View>(
   program: Program<Init, State, Action, View>,
   init: Init,
   render: (view: View) => void,
-  effectManagers: ReadonlyArray<EffectManager<string, unknown, unknown>> = []
+  effectManagers: ReadonlyArray<EffectManager<string, Action, unknown>> = []
```

Comment:
Each effect manager has its own Action type that is separate from the App/Program action that is defined by the application developer.

Reply:
Right! This was plain wrong.